examples/mapbox/dashboard-app/.env

```diff
@@ -1 +1,3 @@
-REACT_APP_MAPBOX_KEY=pk.eyJ1Ijoia2FsaXBzaWsiLCJhIjoiY2p3Z3JrdjQ4MDRjdDQzcGFyeXBlN3ZtZiJ9.miVaze_snePdEvitucFWSQ
\ No newline at end of file
+REACT_APP_MAPBOX_KEY=pk.eyJ1Ijoia2FsaXBzaWsiLCJhIjoiY2p3Z3JrdjQ4MDRjdDQzcGFyeXBlN3ZtZiJ9.miVaze_snePdEvitucFWSQ
+REACT_APP_CUBEJS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2Mjc0NjM2MDZ9.1boj2JrVcsxVkbQsZxuOP21VDxNQyHpxrh3go45k9pc
```

Comment:
<img width="729" alt="Screenshot 2022-07-28 at 14 45 09" src="https://user-images.githubusercontent.com/3852894/181487267-eb9e42c1-5c39-4d9b-b345-9b1c7cf770c1.png">
This is not a long-lasting JWT. Since that's an example, let's use something that will work for years, and make sure it's signed with the well-known key ("SECRET" is suitable for this purpose).

This is a good token: https://github.com/cube-js/cube.js/blob/master/examples/recipes/entity-attribute-value/queries/run.sh#L8